lib/elixir/pages/Compatibility and Deprecations.md

```diff
@@ -1,5 +1,25 @@
 # Compatibility and Deprecations
 
+## Compatibility between Elixir v1.x versions
+
+Elixir v1.x releases are backwards compatible: well-defined behaviours and documented APIs in a given release will continue working on future releases.
+
+Although we expect the vast majority of programs to remain compatible over time, it is impossible to guarantee that no future change will break any program. Under some unlikely circumstances, we may introduce changes that break existing code:
+
+  * Security: a security issue in the implementation may arise whose resolution requires backwards incompatible changes. We reserve the right to address such security issues.
+
+  * Bugs: if an application has undesired behaviour, a program that depends on the buggy behaviour may break if the bug is fixed. We reserve the right to fix such bugs.
```

Comment:
> if an application has undesired behaviour

This is a bit unclear to me. Do you mean an Elixir application (logger, mix) when you say that? Maybe we can rephrase this as:

> Bugs: if your application relies on a bug in Elixir, then your application might break if the bug is fixed. We reserve the right to fix such bugs.